dash/testing/browser.py

```diff
@@ -456,6 +456,9 @@ def visit_and_snapshot(self, resource_path, hook_id):
             self.driver.get(self.server_url + resource_path)
             self.wait_for_element_by_id(hook_id)
             self.percy_snapshot(resource_path)
+            assert not self.driver.find_elements_by_css_selector(
+                "div.dash-debug-alert"
+            ), "devtools should not raise an error alert"
```

Comment:
🏆

Reply:
though, might need an option to disable this in case someone *wants* to snapshot the errors

Reply:
![Screen Shot 2019-09-13 at 11 54 45 PM](https://user-images.githubusercontent.com/1394467/64903158-e4d80b80-d681-11e9-8add-a195f85da7b9.png)
works like a charm